api/v1alpha1/operator_types_test.go

```diff
@@ -8,39 +8,51 @@ import (
 	"io/fs"
 	"strconv"
 	"strings"
-
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"testing"
 
 	"github.com/operator-framework/operator-controller/internal/conditionsets"
+	// replaces with "slices" in go 1.21
```

Comment:
Nit: put this comment on the same line as the import to be more friendly with tooling that sorts/groups imports.